git_hub_links.rb

```diff
@@ -92,15 +93,15 @@ def json
 
     def pull_request_api_url
       addl_params = "\\&access_token\\=#{access_token}" if access_token
```

Comment:
I wonder how the Access Token that we've generated will work w/ non sul-dlss organizations.  This is intended to give use private repo access along w/ not having requests throttled.  It may have the read-access it needs for non sul-dlss repos, but not 100% sure how their permissions work in this respect.

Reply:
Good question, @jkeck. Assumptions follow. I assume this is a personal access token attached to the @sul-devops-team account? If so, as long as we grant that account sufficient access (`write`, I assume) to these repositories outside the sul-dlss organization, I would expect this token to continue working.

Reply:
I'm not 100% sure if the token is attached to the devops account or not.  (maybe 🤷‍♂️)

FWIW, I don't believe this script specifically needs write access (although perhaps the token does have that).  Just something to be aware of.

Reply:
@jkeck Assuming for now the token is attached to the @sul-devops-team account (and whatever access it has), I've invited that account to the LD4P `sinopia` team, which would grant it write access to the relevant repos. Thanks for raising this!

Reply:
This account has accepted the invitation and should now have the access it needs.